polling/frequent/workflow.go

```diff
@@ -0,0 +1,29 @@
+package frequent
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+func FrequentPolling(ctx workflow.Context) (string, error) {
+	logger := workflow.GetLogger(ctx)
+
+	ao := workflow.ActivityOptions{
+		StartToCloseTimeout: 10 * time.Second,
```

Comment:
StartToClose timeout should be set to a very long value as the activity is expected to be as long-running as needed for the resource to become available. 
HeartbeatTimeout must be specified for activity heartbeat to be ever recorded. This also allows restarting an activity quickly after its worker failure.

Reply:
I have adopted the same StartToCloseTimeout of the java example, do you think it should be higher in Java too? https://github.com/temporalio/samples-java/blob/main/src/main/java/io/temporal/samples/polling/frequent/FrequentPollingWorkflowImpl.java#L40

Reply:
Yes, the Java sample should be fixed as well.

Reply:
I have opened https://github.com/temporalio/samples-java/pull/411 for the issue in the Java samples